components/LinkCollective.js

```diff
@@ -32,17 +26,17 @@ const LinkCollective = ({ target, title, noTitle, collective, children, ...props
     return children || collective.name;
   }
 
-  const { type, slug, name, parentCollective, isIncognito } = collective;
+  const { type, slug, name, isIncognito } = collective;
   if (type === 'USER' && (!name || isIncognito || !slug)) {
     return children || <FormattedMessage id="profile.incognito" defaultMessage="Incognito" />;
   }
-  return type !== 'EVENT' ? (
+  return ![CollectiveType.EVENT, CollectiveType.PROJECT].includes(type) ? (
     <Link href={`/${slug}`} {...props} title={noTitle ? null : title || name} target={target}>
       {children || name || slug}
     </Link>
   ) : (
     <Link
-      href={`/${getEventParentCollectiveSlug(parentCollective)}/events/${slug}`}
+      href={`${getCollectivePageRoute(collective)}`}
       title={noTitle ? null : title || name}
       target={target}
       {...props}
```

Comment:
:ice_cream: Looks like this whole thing can be simplified, if `href` is the only changing prop then we don't use the ternary and can directly do:

```jsx
<Link
  href={getCollectivePageRoute(collective)}
  ...
```